okteto-stack.yaml

```diff
@@ -2,7 +2,7 @@ name: fastapi-crud
 services:
   fastapi:
     public: true
-    image: youngestdev/fastapi-crud:latest
+    image: okteto.dev/fastapi-crud:latest
```

Comment:
making this change will affect the first blog post on the series? what do you suggest we do, should be keep it in a different branch?

Reply:
Yes, that's why I asked if you could create a new branch 'bulk-recipes' and merge this change into it.